pom.xml

```diff
@@ -119,6 +119,7 @@
     <slf4j.version>1.7.16</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
     <hadoop.version>2.6.5</hadoop.version>
+    <aws-java-sdk-bundle.version>1.11.563</aws-java-sdk-bundle.version>
```

Comment:
why is this version different?

Reply:
This is the version hadoop 3.1 has pinned. I'm not married to it though. What matters is the right build command since it'll override the version.